org.eclipse.dawnsci.nexus.test/src/org/eclipse/dawnsci/nexus/ComplexNexusFileTest.java

```diff
@@ -73,7 +74,7 @@ private void setupTestData() {
 		testData.put("sourceProbe", "X-ray");
 		testData.put("sourceType", "Synchotron X-Ray Source");
 
-		testData.put("imageNumber", DatasetFactory.createRange(9.0, Dataset.FLOAT64));
+		testData.put("imageNumber", DatasetFactory.createRange(9.0));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "imageNumber" 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)